### PR TITLE
Add fetishnetwork xPath scraper

### PR DESCRIPTION
**Pull request by niemands**
Requested in #73

### CODE REVIEW

File: scrapers/FetishNetwork.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        postProcess:
+          - replace:
+              - regex: Description:\s?
+                with:
+              - regex: �
+                with:
+      Date:
+        selector: //h4[@class="light-grey"]
+        postProcess:
+          - parseDate: Januery 2, 2006
```
Comment (bnkai) [inline scrapers/FetishNetwork.yml:28]: typo

**Comment by bnkai** [comment]
It seems that depending on the `site` the layout changes 
eg `www.fetishnetwork.com/t2/showgal.php?g=content/flash/realbondagevideos/042_jasmyn_1/25078/516_1&s=1252` needs `//div[@class="display-panel"]/img/@src` as an image or `http://www.fetishnetwork.com/t2/showgal.php?g=content/flash/p0018_s0012/p0018_s0012_atad_013b_02_video/20433/510_1` needs more changes...

for the image part keeping this as a fallback after the cover could be enough 
`(//img[contains(@src,"content/flash/")]/@src)[1]` (havent tested fully yet will do that later )

File: scrapers/FetishNetwork.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          - replace:
+              - regex: ^(.+)(?:\s\|).+$
+                with: $1
+      Details:
+        selector: //div[contains(@class,"description-long-text")]//text()
+        postProcess:
+          - replace:
+              - regex: Description:\s?
+                with:
+              - regex: �
```
Comment (bnkai) [inline scrapers/FetishNetwork.yml:23]: Do you by any chance have the url for this scene handy?
I am curious to what `//div[contains(@class,"description-long-text")]` would show

Comment (niemands) [reply scrapers/FetishNetwork.yml:23]: Some older scenes have them, e.g.:
`http://www.fetishnetwork.com/t2/showgal.php?g=content/flash/p0007_s0018/p0007_s0018_0058breastfest_videos/29269/506_1&s=1100&uvar=typein.4.10.10.0.0.0.0.0`

**Pull request merged**